### PR TITLE
feat(feature-flag): add feature flag from gb

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
Previously we were using manual feature flags, we want to adopt growth book (gb) now to use a % based rollout.

## Solution
use growthbook sdk + define constants

### CODE REVIEW

File: src/constants/featureFlags.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const FEATURE_FLAGS = {
+  STYLING_REVAMP: "styles",
```
[the remark this answers is not in the document]
Comment (seaerchin) [reply src/constants/featureFlags.ts:2]: kek jan chose the name HAHAHA but yes, duly noted for next time!

[the remark this answers is not in the document]
Comment (seaerchin) [reply src/constants/featureFlags.ts:2]: also good point, will change!

**Review by QiluXie — APPROVED** [review]
lgtm

File: src/constants/featureFlags.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (seaerchin) [reply src/constants/featureFlags.ts:2]: updated

**Review by harishv7 — APPROVED** [review]

**Pull request merged**